src/support/archive.cpp

```diff
@@ -210,8 +210,8 @@ static uint32_t read32be(const uint8_t* buf) {
 }
 
 void Archive::dump() const {
-  printf("Archive data %p len %zu, firstRegularData %p\n", data.data(),
-         data.size(), firstRegularData);
+	printf("Archive data %p len %zu, firstRegularData %p\n", data.data(),
```

Comment:
Visual Studio editor artifact. I'll have to change Visual Studio indentation settings. Apparently there's no pretty print scheduled? So it's all up to the programmer to manually pretty print?